### ISSUE TITLE
[BUG] adding in SAPLaunchpad entitlement loops infinitely when configuring.

### ISSUE THREAD

**Issue by dachtera** [SAP-samples/btp-setup-automator#256]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest docker image for BTPSA?

- [X] I'm using the latest docker image for BTPSA.

### Which area is mainly impacted

Cloud Foundry setups

### Current Behavior

When attempting to add the SAP Launchpad service with btpsa, the process will loop infinitely until aborted, with no success

### Expected Behavior

Entitlement addition should complete.

### Steps To Reproduce

Using the following in the json parameters, 

```
"services": [
{
            "name": "SAPLaunchpad",
            "plan": "standard",
            "category": "APPLICATION",
            "entitleonly": true
        }
```
This was generated from the type-ahead functionality.

### Anything else?

Output that repeats until manually halted.

```
####################################################################################################
# Track creation of service instances and app subscriptions
####################################################################################################
[2022-10-13 15:48:36] INFO       : check status of app subscriptions
[2022-10-13 15:48:36] RUN COMMAND: btp --format json list accounts/subscription --subaccount '07f9e26d-ed9d-44e8-bcb4-8d19c9da3aaa'
[2022-10-13 15:48:43] INFO       : check status of app subscriptions
[2022-10-13 15:48:43] RUN COMMAND: btp --format json list accounts/subscription --subaccount '07f9e26d-ed9d-44e8-bcb4-8d19c9da3aaa'
[2022-10-13 15:48:50] INFO       : check status of app subscriptions
[2022-10-13 15:48:50] RUN COMMAND: btp --format json list accounts/subscription --subaccount '07f9e26d-ed9d-44e8-bcb4-8d19c9da3aaa'
[2022-10-13 15:48:56] INFO       : check status of app subscriptions
[2022-10-13 15:48:56] RUN COMMAND: btp --format json list accounts/subscription --subaccount '07f9e26d-ed9d-44e8-bcb4-8d19c9da3aaa'
[2022-10-13 15:49:03] INFO       : check status of app subscriptions
[2022-10-13 15:49:03] RUN COMMAND: btp --format json list accounts/subscription --subaccount '07f9e26d-ed9d-44e8-bcb4-8d19c9da3aaa'
[2022-10-13 15:49:09] INFO       : check status of app subscriptions
[2022-10-13 15:49:09] RUN COMMAND: btp --format json list accounts/subscription --subaccount '07f9e26d-ed9d-44e8-bcb4-8d19c9da3aaa'
[2022-10-13 15:49:16] INFO       : check status of app subscriptions
[2022-10-13 15:49:16] RUN COMMAND: btp --format json list accounts/subscription --subaccount '07f9e26d-ed9d-44e8-bcb4-8d19c9da3aaa'
[2022-10-13 15:49:22] INFO       : check status of app subscriptions
[2022-10-13 15:49:22] RUN COMMAND: btp --format json list accounts/subscription --subaccount '07f9e26d-ed9d-44e8-bcb4-8d19c9da3aaa'
[2022-10-13 15:49:29] INFO       : check status of app subscriptions
[2022-10-13 15:49:29] RUN COMMAND: btp --format json list accounts/subscription --subaccount '07f9e26d-ed9d-44e8-bcb4-8d19c9da3aaa'
[2022-10-13 15:49:35] INFO       : check status of app subscriptions
[2022-10-13 15:49:35] RUN COMMAND: btp --format json list accounts/subscription --subaccount '07f9e26d-ed9d-44e8-bcb4-8d19c9da3aaa'
[2022-10-13 15:49:42] INFO       : check status of app subscriptions
[2022-10-13 15:49:42] RUN COMMAND: btp --format json list accounts/subscription --subaccount '07f9e26d-ed9d-44e8-bcb4-8d19c9da3aaa'
```

**Comment by lechnerc77** [comment]
Update on status:
- Error is located in file `btpcli.py`, function `track_creation_of_subscriptions_and_services`.
- The services called in the subsequent check methods do not respect the setting `entitleonly`. The check must be executed only for services and supscriptions that should be created.

**Comment by lechnerc77** [comment]
This bug is fixed with PR #258 . A new Docker image is available in GitHub packages that contains the fix
